packages/connectivity/README.md

```diff
@@ -48,6 +50,31 @@ dispose() {
 }
 ```
 
+Additional methods available:
+
+```dart
+@override
+initState() {
+  super.initState();
+
+  await Connectivity().getWifiBSSID(); // obtains the BSSID of the connected wifi network
+  await Connectivity().getWifiIP();    // obtains the IP address of the connected wifi network
+  await Connectivity().getWifiName();  // obtains the wifi name (SSID) of the connected wifi network
+}
+```
+
+### Known Issues
+
+#### iOS 13
+
+The methods `.getWifiBSSID()` and `.getWifiName()` utilize the [CNCopyCurrentNetworkInfo](https://developer.apple.com/documentation/systemconfiguration/1614126-cncopycurrentnetworkinfo) function on iOS.
+
+As of iOS 13 that api will no longer return valid information by default and will instead return the following:
+> SSID: "Wi-Fi" or "WLAN" ("WLAN" will be returned for the China SKU)  
+> BSSID: "00:00:00:00:00:00"
+
+You can follow issue [#37804](https://github.com/flutter/flutter/issues/37804) for the changes required to return valid SSID and BSSID values with iOS 13.
```

Comment:
We should just explain how can these work in detail. like what was explained from Apple's email.
And I think one of the options to make this work is to request user's location service permission. So let's add that to the doc as well. 
You can find an example here in the image picker plugin's readme asking developer to put necessary keys into the info.plist.
https://github.com/flutter/plugins/blob/master/packages/image_picker/README.md

Reply:
>And I think one of the options to make this work is to request user's location service ?>permission. So let's add that to the doc as well.
>You can find an example here in the image picker plugin's readme asking developer to >put necessary keys into the info.plist.
>https://github.com/flutter/plugins/blob/master/packages/image_picker/README.md

I think we should do this with the PR that resolves https://github.com/flutter/flutter/issues/37804. So nothing needs to be done here.

Reply:
Won't be done now. Resolving